### PR TITLE
[ko] Fixed translation error in apiserver-aggregation page

### LINKED ISSUE #34667: [ko] Fix a translation error in docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/
**This is a Bug Report**

**Problem:**
애그리게이션 레이어는 [사용자 정의 리소스](https://kubernetes.io/ko/docs/concepts/extend-kubernetes/api-extension/custom-resources/)와는 다르며, 애그리게이션 레이어는 [kube-apiserver](https://kubernetes.io/ko/docs/concepts/overview/components/#kube-apiserver) 가 새로운 종류의 오브젝트를 인식하도록 하는 방법이다.

**Proposed Solution:**
애그리게이션 레이어는 [kube-apiserver](https://kubernetes.io/ko/docs/concepts/overview/components/#kube-apiserver) 가 새로운 종류의 오브젝트를 인식하도록 하는 방법인 [사용자 정의 리소스](https://kubernetes.io/ko/docs/concepts/extend-kubernetes/api-extension/custom-resources/)와는 다르다.

**Page to Update:**
https://kubernetes.io/ko/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/

### PR DESCRIPTION
**Pull request by bishal7679**

### CODE REVIEW

File: content/ko/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ weight: 10
 애그리게이션 레이어는 코어 쿠버네티스 API가 제공하는 기능 이외에 더 많은 기능을 제공할 수 있도록 추가 API를 더해 쿠버네티스를 확장할 수 있게 해준다.
 추가 API는 [메트릭 서버](https://github.com/kubernetes-sigs/metrics-server)와 같이 미리 만들어진 솔루션이거나 사용자가 직접 개발한 API일 수 있다.
 
-애그리게이션 레이어는 [사용자 정의 리소스](/ko/docs/concepts/extend-kubernetes/api-extension/custom-resources/)와는 다르며, 애그리게이션 레이어는 {{< glossary_tooltip term_id="kube-apiserver" text="kube-apiserver" >}} 가 새로운 종류의 오브젝트를 인식하도록 하는 방법이다.
+애그리게이션 레이어는 {{< glossary_tooltip term_id="kube-apiserver" text="kube-apiserver" >}} 가 새로운 종류의 오브젝트를 인식하도록 하는 방법인 [사용자 정의 리소스](/ko/docs/concepts/extend-kubernetes/api-extension/custom-resources/)와는 다르며.
```
Comment (jihoon-seo) [inline content/ko/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md:16]: From #34667:

```suggestion
애그리게이션 레이어는 {{< glossary_tooltip term_id="kube-apiserver" text="kube-apiserver" >}}가 새로운 종류의 오브젝트를 인식하도록 하는 방법인 [사용자 정의 리소스](/ko/docs/concepts/extend-kubernetes/api-extension/custom-resources/)와는 다르다.
```

**Comment by seokho-son** [comment]
Hi @bishal7679 ,
I am one of the approvers for Korean localization.

Issue #34667 was opened by @henjoon 
We generally assume that the author of Issue has a high priority to be the assignee.

Since you've self-assigned for the issue #34667 , I hope to check @henjoon has willing to resolve this issue first.
@bishal7679 Sorry for inconvenience but this kind of workflow is important for opensource project. :) 

(cc @jihoon-seo we need to coordinate PR before we initiate the review.)

**Comment by bishal7679** [comment]
@seokho-son Can you review it again ?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Comment by tengqm** [comment]
/retitle [ko] Fixed translation error in apiserver-aggregation page

**Comment by seokho-son** [comment]
/approve

Thanks !

**Pull request merged**